### PR TITLE
Corrections in docs and tools after releasing provider RCs

### PR DESCRIPTION
**Pull request by potiuk**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Comment by potiuk** [comment]
Round of updates after releasing providers RC1 regular, backports and the documentation.

**Review by kaxil — COMMENTED** [review]
Let's wait till we release them -- incase we need to fix a provider  -- we might have to change

**Comment by potiuk** [comment]
Sure. no hurry with that.

File: dev/README_RELEASE_PROVIDER_PACKAGES.md [hunk 1]
```diff
@@ -278,26 +278,42 @@ cd "${AIRFLOW_REPO_ROOT}"
 - Now you can preview the documentation.
 
 ```shell script
-./docs/start_doc_server.sh
+./docs/start_doc_server1.sh
```
Comment (mik-laj) [inline dev/README_RELEASE_PROVIDER_PACKAGES.md]: ?

File: dev/README_RELEASE_PROVIDER_PACKAGES.md [hunk 2]
```diff
@@ -321,7 +337,7 @@ EOF
 cat <<EOF
 Hey all,
 
-I have cut Airflow Providers. This email is calling a vote on the release,
+I have just cut the new wave f Airflow Providers packages. This email is calling a vote on the release,
```
Comment (mik-laj) [inline dev/README_RELEASE_PROVIDER_PACKAGES.md]:
```suggestion
I have just cut the new wave Airflow Providers packages. This email is calling a vote on the release,
```

File: dev/README_RELEASE_PROVIDER_PACKAGES.md (hunk 1 above)
Comment (potiuk) [reply dev/README_RELEASE_PROVIDER_PACKAGES.md]: No idea :)

Comment (potiuk) [inline dev/README_RELEASE_PROVIDER_PACKAGES.md]:
```suggestion
./docs/start_doc_server.sh
```

File: dev/README_RELEASE_PROVIDER_PACKAGES.md [hunk 3]
```diff
@@ -283,21 +283,37 @@ cd "${AIRFLOW_REPO_ROOT}"
...
 cd "${AIRFLOW_SITE_DIRECTORY}"
 ```
 
+If you have released just a few providers:
+
+```shell script
+./docs/publish_docs.py \
+    --package-filter apache-airflow-providers \
+  --package-filter 'apache-airflow-providers-PACKAGE1' \
+  --package-filter 'apache-airflow-providers-PACKAGE2' \
```
Comment (kaxil) [inline dev/README_RELEASE_PROVIDER_PACKAGES.md]: nit
```suggestion
./docs/publish_docs.py \
    --package-filter apache-airflow-providers \
    --package-filter 'apache-airflow-providers-PACKAGE1' \
    --package-filter 'apache-airflow-providers-PACKAGE2' \
```

**Review by kaxil — APPROVED** [review]

**Pull request merged**